codebreaker_console/helpers/validations.rb

```diff
@@ -0,0 +1,17 @@
+module Validations
+  def valid_name?(username)
```

Comment:
It's incorrect validation when you create it just for one field of some class and move it to separate method